src/lib/components/event/event-summary-row.svelte

```diff
@@ -138,9 +138,7 @@
   <td
     class="w-full overflow-hidden text-right text-sm font-normal xl:text-left"
   >
-    <div
-      class="flex w-full max-w-screen-sm items-center gap-2 lg:max-w-screen-md xl:max-w-screen-xl"
-    >
+    <div class="flex w-full items-center gap-2">
       <Icon name={icon} />
```

Comment:
Might want to add a `shrink-0` here
```suggestion
      <Icon name={icon} class="shrink-0" />
```

| Before | After |
|-|-|
|![Screenshot 2024-10-02 at 3 03 07 PM](https://github.com/user-attachments/assets/44d7b212-79b9-4725-aeec-0e83ac6d4ff9)|![Screenshot 2024-10-02 at 3 02 53 PM](https://github.com/user-attachments/assets/ae93c5a4-8a79-4d78-800f-04fb7f86754f)|

Reply:
Maybe I should add that on that Icon component itself?